openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -308,37 +308,19 @@ def create_audioset_view_query():
         f"""
         CREATE VIEW public.{AUDIOSET_VIEW_NAME}
         AS
-            SELECT DISTINCT
-                cast(
-                    audio_set ->> 'foreign_identifier'  as varchar(1000)
-                ) as foreign_identifier,
-                cast(
-                    audio_set ->> 'title'               as varchar(2000)
-                ) as title,
-                cast(
-                    audio_set ->> 'foreign_landing_url' as varchar(1000)
-                ) as foreign_landing_url,
-                cast(
-                    audio_set ->> 'creator'             as varchar(2000)
-                ) as creator,
-                cast(
-                    audio_set ->> 'creator_url'         as varchar(2000)
-                ) as creator_url,
-                cast(
-                    audio_set ->> 'url'                 as varchar(1000)
-                ) as url,
-                cast(
-                    audio_set ->> 'filesize'            as integer
-                )       as filesize,
-                cast(
-                    audio_set ->> 'filetype'            as varchar(80)
-                )   as filetype,
-                cast(
-                    audio_set ->> 'thumbnail'           as varchar(1000)
-                ) as thumbnail,
-                provider
-            FROM public.{AUDIO_VIEW_NAME}
-            WHERE audio_set IS NOT NULL;
+        SELECT DISTINCT
+            audio_set ->> 'foreign_identifier'  :: varchar(1000) :: foreign_identifier,
```

Comment:
I think these may all need to be in the following format (I talk about why a bit in https://github.com/WordPress/openverse-api/pull/648):
```suggestion
            (audio_set ->> 'foreign_identifier')  :: varchar(1000) as foreign_identifier,
```

Reply:
Sorry about that, clearly I didn't read far enough. Fixed and re-tested.